plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsMaker.java

```diff
@@ -124,6 +126,9 @@ private TableStatistics makeTableStatistics(IcebergTableHandle tableHandle)
             if (summary.getColumnSizes() != null) {
                 Long columnSize = summary.getColumnSizes().get(fieldId);
                 if (columnSize != null) {
+                    if (columnHandle.getBaseType().equals(VarcharType.VARCHAR) || columnHandle.getBaseType().equals(VarbinaryType.VARBINARY)) {
+                        columnSize = columnSize * 10;
```

Comment:
Can we leave a code comment about why multiplying 10?

Reply:
Sure thing, sorry I didn't do it in the first place